doc/tutorials/content/gasd_estimation.rst

```diff
@@ -99,6 +99,13 @@ The following code snippet will estimate a GASD shape + color descriptor for an
 
      // Get the alignment transform
      Eigen::Matrix4f trans = gasd.getTransform (trans);
+
+     // descriptor.points.size () should be of size 1
+     // Unpack histogram bins
+     for (size_t i = 0; i < size_t( descriptor.points[0].descriptorSize ()); ++i)
+     {
+       descriptor.points[0].histogram[i];
```

Comment:
Can replace all instances of `descriptor.points[0]` with `descriptor[0]`. We might hide access to `points` in the future.